examples/react-router/components/Nav.js

```diff
@@ -7,13 +7,14 @@ import {Link} from 'react-router';
 
 class Nav extends React.Component {
     static contextTypes = {
-        router: React.PropTypes.func.isRequired
+        history: React.PropTypes.object.isRequired,
+        location: React.PropTypes.object.isRequired
     };
     render() {
         return (
             <ul className="pure-menu pure-menu-open pure-menu-horizontal">
-                <li className={this.context.router.isActive('/') ? 'pure-menu-selected' : ''}><Link to='/'>Home</Link></li>
-                <li className={this.context.router.isActive('/about') ? 'pure-menu-selected' : ''}><Link to='/about'>About</Link></li>
+                <li className={this.context.location.pathname === '/' ? 'pure-menu-selected' : ''}><Link to='/'>Home</Link></li>
```

Comment:
So the issue I saw was that `/about` was nested under `/`, so when I was on the about page both links were highlighted. To prevent this, I'm checking the specific route for the home page. I'm not sure if there's a way to fix this in the `routes.jsx` or not.